### ISSUE TITLE
Fetch updates

### ISSUE THREAD

**Issue by polkaulfield** [kassindornelles/lol-for-linux-installer#32]
It would be nice to fetch the latest launcher script from the repo without reinstalling, because of the acrtive development. Also listing the wine versions available would be pretty nice. 

Another idea is to bundle winetricks directly, i plan to add it into the appimage or we could fetch it from the repo itself at installation.

**Comment by kassindornelles** [comment]
Check: https://github.com/kassindornelles/lol-for-linux-installer/issues/29#issuecomment-1508893890

Also the appimage might need some changes since we copy the launcher.py files instead of creating them inside the main script: https://github.com/kassindornelles/lol-for-linux-installer/tree/main/python_src/src

**Comment by kassindornelles** [comment]
> It would be nice to fetch the latest launcher script from the repo without reinstalling, because of the acrtive development. Also listing the wine versions available would be pretty nice.
> 
> Another idea is to bundle winetricks directly, i plan to add it into the appimage or we could fetch it from the repo itself at installation.

Yes, this would be much better than using system winetricks, is breaking for some people on old distros

**Comment by kassindornelles** [comment]
I also want to move all the code from the leagueinstaller.py to the main "Install league.py" file bc we are having problems with the way the tell the second script about new options the users set in the UI

for example, now we pass 5 arguments to the script and thats the limit as far as i know, i cant add a option for nvidia hybrid users because of that limitation

**Comment by polkaulfield** [comment]
We can pass an object with the parameters xD But yeah this needs some reestructuring before adding new features or we'll end up with a mess.

**Comment by kassindornelles** [comment]
btw, how do i build the appimage?
wanted to make a release with appimage but i have no experience with it :p

**Comment by kassindornelles** [comment]
also a really basic wine-build update system is in place:
https://github.com/kassindornelles/lol-for-linux-installer/commit/7e106de548a92575eee0d0627128c7ed04442332

**Comment by kassindornelles** [comment]
not gonna happen since we are packaging it to common formats now... new version > new updated package, maybe a repository for debian/ubuntu and AUR for arch
